src/components/Attributes/AttributeRow.tsx

```diff
@@ -193,6 +200,24 @@ const AttributeRow: React.FC<AttributeRowProps> = ({
           />
         </BasicAttributeRow>
       );
+    case AttributeInputTypeEnum.BOOLEAN:
+      return (
+        <BasicAttributeRow label={attribute.label}>
+          <div className={classes.pullRight}>
+            <Checkbox
+              disabled={disabled}
+              name={`attribute:${attribute.label}`}
+              onChange={event =>
+                onChange(attribute.id, JSON.stringify(event.target.checked))
+              }
+              checked={JSON.parse(attribute.value[0] ?? "false")}
```

Comment:
Why are we stringifying boolean values here?

Reply:
TS is crying... 😢